src/piskel-boot.js

```diff
@@ -9,7 +9,11 @@
     version = '';
   }
 
-  window.onPiskelReady = function () {
+  if (!window.onPiskelReady) {
```

Comment:
nit: use a name that hints at an array? "piskelReadyCallbacks" or something like that.